src/client/api/quoteBundleIsBundleSelector.ts

```diff
@@ -0,0 +1,4 @@
+import { QuoteBundle } from 'data/graphql'
+
+export const quoteBundleIsBundleSelector = (bundle: QuoteBundle) =>
```

Comment:
The word `Selector` in this name is a little confusing to me. How about just `quoteBundleIsBundle` or something like `quoteBundleHasMultipleQuotes` or `quoteBundleIsTrueBundle`?